README.md

```diff
@@ -1,50 +1,86 @@
-Yojson: low-level JSON library for OCaml
-========================================
+Yojson: JSON library for OCaml
+==============================
 
 [![Build Status](https://travis-ci.org/ocaml-community/yojson.svg?branch=master)](https://travis-ci.org/ocaml-community/yojson)
 
-_This library is for manipulating the json AST directly. For mapping between OCaml types and json, we recommend [atdgen](https://github.com/ahrefs/atd)._
+This library parses complete JSON files into a nested OCaml data structure,
+similarly to DOM parsers for XML. For stream processing in a SAX-like fashion
```

Comment:
I'm personally not familiar with SAX so maybe a link would help people like me!

Reply:
Also Yojson supports parsing streams of JSON values. It's two different things but it might be worth clarifying this to avoid scaring of users. I'm wondering if the sentence `parses complete JSON files into a nested OCaml data structure` doesn't narrow the scope of the library a bit too much.

Reply:
I've reformulated it a bit and moved the mention of `jsonm` to the "related" section; you're right there is no point in dragging XML concepts into JSON.